src/avrenderer/aardvark_renderer.cpp

```diff
@@ -285,15 +299,24 @@ void VulkanExample::renderSceneToTarget( uint32_t cbIndex, vks::RenderTarget tar
 
 void VulkanExample::renderScene( uint32_t cbIndex, VkRenderPass targetRenderPass, VkFramebuffer targetFrameBuffer, uint32_t targetWidth, uint32_t targetHeight, EEye eEye )
 {
-
+	const VkClearColorValue k_vkClearColorDefault = { 0.0f, 0.0f, 0.0f, 0.0f };
 	VkClearValue clearValues[3];
+	VkClearColorValue clearColor = k_vkClearColorDefault;
+	if (eEye == EEye::Mirror && m_rendererConfig.m_mixedRealityEnabled) {
+		clearColor = {
+			m_rendererConfig.m_clearColor[0],
+			m_rendererConfig.m_clearColor[1],
+			m_rendererConfig.m_clearColor[2],
+			1.0f };
+	}
```

Comment:
clearColor could be made more generic here and allowed all the time, not sure anything but transparent black really makes sense unless your intending mixed reality capture.